blog/docs/articles/knative-serving-in-k0s.md

```diff
@@ -0,0 +1,436 @@
+```
+
+```bash
+NAME                                      CPU-REQUEST   CPU-LIMIT   MEM-REQUEST   MEM_LIMIT
+activator-7499d967fc-2npcf                300m          1           60Mi          600Mi
+autoscaler-568989dd8c-qzrhc               100m          1           100Mi         1000Mi
+autoscaler-hpa-854dcfbd44-8vcj8           30m           300m        40Mi          400Mi
+controller-76c798ffcb-k96sz               100m          1           100Mi         1000Mi
+default-domain-nwbhr                      100m          1           100Mi         1000Mi
+domain-mapping-7748ff49d4-29mg4           30m           300m        40Mi          400Mi
+domainmapping-webhook-755d864f5c-dsc7j    100m          500m        100Mi         500Mi
+net-kourier-controller-79c998474f-svzcm   <none>        <none>      <none>        <none>
+webhook-8466d59795-d8zd8                  100m          500m        100Mi         500Mi
+```
+
+kourier-system
+
+```bash
+kubectl get pods  -n kourier-system -o custom-columns="NAME:metadata.name,CPU-REQUEST:spec.containers[*].resources.requests.cpu,CPU-LIMIT:spec.containers[*].resources.limits.cpu,MEM-REQUEST:spec.containers[*].resources.requests.memory,MEM_LIMIT:spec.containers[*].resources.limits.memory"
+```
+```bash
+NAME                                      CPU-REQUEST   CPU-LIMIT   MEM-REQUEST   MEM_LIMIT
+3scale-kourier-gateway-5f9f97b454-rqkgh   <none>        <none>      <none>        <none>
+```
+
+
+
+**Total**
+
+
+| **Component**          | **CPU request** | **CPU Limit** | **Memory Request** | **Memory Limit** |
+|------------------------|-----------------|---------------|--------------------|------------------|
+| activator              | 300m            | 1             | 60Mi               | 600Mi            |
+| autoscaler             | 100m            | 1             | 100Mi              | 1000Mi           |
+| autoscaler-hpa         | 30m             | 300m          | 40Mi               | 400Mi            |
+| controller             | 100m            | 1             | 100Mi              | 1000Mi           |
+| default-domain*        | 100m            | 1             | 100Mi              | 1000Mi           |
+| domain-mapping         | 30m             | 300m          | 40Mi               | 400Mi            |
+| domainmapping-webhook  | 100m            | 500m          | 100Mi              | 500Mi            |
+| net-kourier-controller | \<none\>        | \<none\>      | \<none\>           | \<none\>         |
+| webhook                | 100m            | 500m          | 100Mi              | 500Mi            |
+| **Total**              | **860m**        | **5600m**     | **640Mi**          | **5400Mi**       |
+
+>\*  default-domain is a Job, and the resource will be freed up once the job completes
```

Comment:
may want to use an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) here